### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by fdiblen**
- Adds link to cffconvert repo
- Adds an example to show how to do validation for subdirectories
- Adds an example to show how to convert to zenodo.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-GitHub action to validate CITATION.cff files, and convert to other citation formats.
+GitHub action to validate CITATION.cff files, and convert to other citation formats using [cffconvert](https://github.com/citation-file-format/cff-converter-python).
 
 ## Badges
 
 | Description | Badge |
 | --- | --- |
 | Zenodo | [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3993241.svg)](https://doi.org/10.5281/zenodo.3993241) |
 | Testing | [![testing](https://github.com/citation-file-format/cffconvert-github-action/workflows/selftest/badge.svg)](https://github.com/citation-file-format/cffconvert-github-action/actions?query=workflow%3A%22selftest%22) |
+| Markdown links | [![Check Markdown links](https://github.com/citation-file-format/cffconvert-github-action/actions/workflows/link-check.yml/badge.svg)](https://github.com/citation-file-format/cffconvert-github-action/actions/workflows/link-check.yml) |
```
Comment (abelsiqueira) [inline README.md:11]: Doesn't exit yet

File: README.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
 # cffconvert GitHub Action
 
-GitHub action to validate CITATION.cff files, and convert to other citation formats.
+GitHub action to validate CITATION.cff files, and convert to other citation formats using [cffconvert](https://github.com/citation-file-format/cff-converter-python).
```
Comment (jspaaks) [inline README.md]: I prefer to link to pypi, and maybe change the text to include "dockerized version of cffconvert"

File: README.md [hunk 3]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+   jobs:
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+        - name: Validate a CITATION.cff from a subdirectory
+          uses: ./
```
Comment (jspaaks) [inline README.md]: use marketplace link

File: README.md [hunk 4]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+        - name: Convert CITATION.cff to Zenodo metadata format
+          id: cffconvert
+          uses: ./
```
Comment (jspaaks) [inline README.md]: marketplace link

File: README.md [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
 | Description | Badge |
 | --- | --- |
 | Zenodo | [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3993241.svg)](https://doi.org/10.5281/zenodo.3993241) |
 | Testing | [![testing](https://github.com/citation-file-format/cffconvert-github-action/workflows/selftest/badge.svg)](https://github.com/citation-file-format/cffconvert-github-action/actions?query=workflow%3A%22selftest%22) |
+| Markdown links | [![Check Markdown links](https://github.com/citation-file-format/cffconvert-github-action/actions/workflows/link-check.yml/badge.svg)](https://github.com/citation-file-format/cffconvert-github-action/actions/workflows/link-check.yml) |
 
+## Usage
 
-## Example usage
+1. Save the one of the snippets as ``.github/workflows/cffconvert.yml``
```
Comment (abelsiqueira) [inline README.md]:
```suggestion
1. Save one of the snippets as ``.github/workflows/cffconvert.yml``
```

File: README.md [hunk 6]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+
+   jobs:
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+        - name: Convert CITATION.cff to Zenodo metadata format
```
Comment (jspaaks) [inline README.md]: is the indent off?

File: README.md [hunk 7]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+          id: cffconvert
+          uses: ./
+          with:
+            args: "--infile ./CITATION.cff -f zenodo > zenodo.json"
```
Comment (jspaaks) [inline README.md]: .zenodo.json

File: README.md [hunk 8]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+            
+        - name: Commit and push Zenodo metadata 
+          run: |
+            git config --global user.name 'cffconvert GitHub Action'
+            git config --global user.email 'cffconvert@users.noreply.github.com'
+            git add zenodo.json
```
Comment (jspaaks) [inline README.md]: .zenodo.json

File: README.md (hunk 7 above)
Comment (jspaaks) [inline README.md]: maybe instead of the short -f use --format

File: README.md [hunk 9]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+            git commit -am "Automated of Zenodo metadata"
```
Comment (jspaaks) [inline README.md]: not sure if the -a is needed, just -m is enough

**Review by jspaaks — CHANGES_REQUESTED** [review]
couple minor things

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
I made a few comments. Jurrian made more.

File: README.md (hunk 2 above)
Comment (fdiblen) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (fdiblen) [reply README.md:11]: it was added in the previous PR

File: README.md (hunk 5 above)
Comment (fdiblen) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (fdiblen) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (fdiblen) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (fdiblen) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (fdiblen) [reply README.md]: done

Comment (fdiblen) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (fdiblen) [reply README.md]: done

File: README.md [hunk 10]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+         - name: Validate a CITATION.cff from a subdirectory
+           uses: citation-file-format/cffconvert-github-action@2.0.0
+           with:
+             args: "--infile ./subdirectory/CITATION.cff --validate"
```
Comment (abelsiqueira) [inline README.md]:
```suggestion
             args: "--infile ./tests/subdirectory/CITATION.cff --validate"
```

File: README.md [hunk 11]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+           with:
+             args: "--infile ./CITATION.cff --format zenodo > .zenodo.json"
+            
+        - name: Commit and push Zenodo metadata 
+          run: |
+            git config --global user.name 'cffconvert GitHub Action'
+            git config --global user.email 'cffconvert@users.noreply.github.com'
+            git add .zenodo.json
+            git commit -m "Automated update of Zenodo metadata"
+            git push
```
Comment (abelsiqueira) [inline README.md]: Bad indentation

File: README.md [hunk 12]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+   jobs:
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+         - name: Convert CITATION.cff to Zenodo metadata format
+           id: cffconvert
```
Comment (abelsiqueira) [inline README.md]:
```suggestion
```
Not sure what it does

File: README.md (hunk 11 above)
Comment (abelsiqueira) [inline README.md]: It this enough? Perhaps it is better to use another action to create a pull request if there is a difference?

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
I noticed some extra issues.

File: README.md [hunk 13]
```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
...
+           uses: citation-file-format/cffconvert-github-action@2.0.0
+           with:
+             args: "--infile ./CITATION.cff --format zenodo > .zenodo.json"
```
Comment (abelsiqueira) [inline README.md]: I tried running the examples with act (see #40) and this line fails with unknown command. The change below works:
```suggestion
             args: "--infile ./CITATION.cff --format zenodo --outfile .zenodo.json"
```

Comment (jspaaks) [reply README.md]: outfile is safer for environments where you may not have redirection, e.g. Windows

File: README.md (hunk 12 above)
Comment (jspaaks) [reply README.md]: `id` can be helpful if you later want to get some information about a step, for example a varibale that was set during the step. You can do something along the lines of `steps.<the id>.outputs.<the var>`. But agree it isnt needed here as we arent using its result.

File: README.md (hunk 10 above)
Comment (jspaaks) [reply README.md]: Doesn't need `tests`, it is just an example of the syntax that users would use in their own repos.

**Review by jspaaks — APPROVED** [review]
Nice work. Are we sure we don;t need to do something with GITHUB_TOKEN in the last example. Something like 

https://github.com/zenodraft/action/blob/f12ee68b00471eb963c651051112b5fac604620e/.github/workflows/publishing.yml#L17

File: README.md (hunk 11 above)
Comment (fdiblen) [reply README.md]: using the web editor is a bad idea.

File: README.md (hunk 12 above)
Comment (fdiblen) [reply README.md]: it is a left over from some tests. I will remove it.

File: README.md (hunk 11 above)
Comment (fdiblen) [reply README.md]: done

Comment (fdiblen) [reply README.md]: a good suggestion but this should be added in another PR

**Comment by fdiblen** [comment]
> Nice work. Are we sure we don;t need to do something with GITHUB_TOKEN in the last example. Something like
> 
> https://github.com/zenodraft/action/blob/f12ee68b00471eb963c651051112b5fac604620e/.github/workflows/publishing.yml#L17

We were also doing the same here

https://github.com/NLeSC/python-template/blob/7e37c487687a10ebe5f9c2296d49f0de9412f67f/%7B%7Bcookiecutter.directory_name%7D%7D/.github/workflows/next_steps.yml#L52

**Review by abelsiqueira — APPROVED** [review]
Thanks, looks good to me.

**Pull request merged**